tests/kernel_config_manager_test.sh

```diff
@@ -100,6 +102,19 @@ function test_save_config_file_check_save_failures()
   assert_equals_helper "Should return ENOENT, because '.config' != '.configuration'" "$LINENO" "$?" "2"
   rm .configuration
 
+  # Test git commit fail scenario
+  touch .config
+  git_name=$(git config user.name)
+  git_email=$(git config user.email)
+  git config --global user.name ""
+  git config --global user.email ""
+  ret=$(save_config_file "$NO_FORCE $NAME_1" "$DESCRIPTION_1")
```

Comment:
Add newline after ret

Reply:
Some of these tests should be on the previous commit, and as previously stated this commit seems unnecessary considering #723